src/test/java/com/pusher/client/channel/impl/PresenceChannelImplTest.java

```diff
@@ -107,6 +107,27 @@ public void testInternalSubscriptionSucceededMessageIsTranslatedToASubscriptionS
     }
 
     @Test
+    public void testMemberAddedMessageLeadsToUserIdBeingStoredCorrectly() {
```

Comment:
I don't think the test name identifies what you're testing? Are we testing that an Integer value for `user_id` in a JSON payload is converted to a string?